fv3config/fv3run/_native.py

```diff
@@ -105,13 +117,9 @@ def _log_exceptions(localdir):
         raise e
 
 
-def _run_experiment(dirname, n_processes, runfile_name=None, mpi_flags=None):
+def _run_experiment(dirname, n_processes, python_args, mpi_flags=None):
     if mpi_flags is None:
         mpi_flags = []
-    if runfile_name is None:
-        python_args = ["python3", "-m", "mpi4py", "-m", "fv3gfs.run"]
-    else:
-        python_args = ["python3", "-m", "mpi4py", runfile_name]
```

Comment:
Related to other comment, this code could be replaced with `python_args = _get_python_args(runfile_name)` without needing any signature changes.

Reply:
I think you commented on some deleted code.

Reply:
I did, but it was intentional.